client/state/help/selectors.js

```diff
@@ -25,14 +25,17 @@ export const getHelpSiteId = state => state.help.selectedSiteId;
 export const getHelpSelectedSite = state => {
 	const siteId = getHelpSiteId( state ) || getSelectedOrPrimarySiteId( state );
 	const helpSite = getSite( state, siteId );
+	if ( helpSite ) {
+		return helpSite;
+	}
 	// Are sites loaded but the help site is not available? We may have a bad site or primary.
 	const sites = get( state, 'sites.items' );
 	const siteKeys = sites && Object.keys( sites );
-	if ( ! helpSite && siteKeys && siteKeys.length > 0 ) {
+	if ( siteKeys && siteKeys.length > 0 ) {
```

Comment:
so while this is awesome because I had a small nitpick and you actually created a second PR, I'm going to propose another nit 🙃 

```js
const sites = Object.keys( get( state, 'sites.items', {} ) );

return sites.length
	? getSite( state, sites[ 0 ] )
	: null;
```

the ternary isn't important. the nit was that we can further reduce this code and push early-abort. without the ternary…

```js
const sites = Object.keys( get( state, 'sites.items', {} ) );

if ( ! sites.length ) {
	return null;
}

return getSite( state, sites[ 0 ] );
```

Reply:
updated in 297a151. Note that state.sites.items we expect to be null in some cases, so the get fallback doesn't cover the falsey cases appropriately.